### PR TITLE
Scenario/C1759

### PR DESCRIPTION
**Pull request by mabalderramav**

### CODE REVIEW

File: IDT.Pinless/Main/UI/Components/NotifyComponent.cs [hunk 1]
```diff
@@ -23,6 +25,9 @@ public class NotifyComponent : BasePage
         public void ClickOnBtnClose()
         {
             CommonActions.ClickElementByJavaScript(BtnClose);
+
+            // Wait until the notify is disappear.
+            Wait.Until(ExpectedConditions.InvisibilityOfElementLocated(By.CssSelector("div.ui-pnotify")));
```
Comment (wMercadoJala) [inline IDT.Pinless/Main/UI/Components/NotifyComponent.cs]: We should move this to the core

File: IDT.Pinless/Main/UI/Pages/PinlessPages/PinlessDialPage.cs [hunk 2]
```diff
@@ -50,55 +49,25 @@ public class PinlessDialPage : BasePage
...
-                DeleteFavoriteNumber(cssSelector);
-
-                // Find again the number of numbers to exist in Favorite numbers section.
-                listBtnDeleteFavoriteNumbers = CommonActions.GetListWebElements(By.CssSelector(cssSelector));
-            }
-        }
 
         /// <summary>
-        /// Deletes to favorite number.
+        /// Delete a favorite number of the list.
```
Comment (wMercadoJala) [inline IDT.Pinless/Main/UI/Pages/PinlessPages/PinlessDialPage.cs]: change verb to third person: Deletes

Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Pages/PinlessPages/PinlessDialPage.cs]: Done

File: IDT.Pinless/Main/UI/Components/NotifyComponent.cs (hunk 1 above)
Comment (mabalderramav) [reply IDT.Pinless/Main/UI/Components/NotifyComponent.cs]: I sent to PR#24 with this logic in the core project. You could review please.

**Pull request merged**